pkg/client/clientset_generated/release_1_1/clientset.go

```diff
@@ -57,18 +58,18 @@ func NewForConfig(c *unversioned.Config) (*Clientset, error) {
 	var err error
 	clientset.LegacyClient, err = legacy_unversioned.NewForConfig(c)
 	if err != nil {
-		return nil, err
+		return &clientset, err
 	}
 	clientset.ExtensionsClient, err = extensions_unversioned.NewForConfig(c)
 	if err != nil {
-		return nil, err
+		return &clientset, err
```

Comment:
We let caller of this function to decide if to fail on error, because it's possible that `Extensions` are disabled through KUBE_API_VERSIONS, but the user still wants to use other groups' clients of the clientset.